library/stack.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+/** @TODO add this to all methods
+ * Push and pop from a stack
+ * @param {string} value
+ * @param {number} value
+ * @return {array}
+ */
+
+class Stack {
+    //  creating a stack of length = lengthOfStack
+    // creating a new array of the length provided
+    // setting up initial index to 0
+    constructor(lengthOfStack) {
+        this.lengthOfStack = this.checkLengthInput(lengthOfStack);
+        this.position = 0;
+        this.stack = new Array(this.lengthOfStack);
+    }
+
+    checkLengthInput(lengthOfStack){
```

Comment:
if you are calling the function check then it should return true or false no ? its returning number